src/Controllers/GagePageController.ts

```diff
@@ -536,13 +581,300 @@ module StreamStats.Controllers {
+                    tooltip: {
+                        headerFormat:'<b>Daily Flow</b>',
+                        pointFormatter: function(){
+                            if (this.formattedPeakDates !== null){
+                                let UTCday = this.x.getUTCDate();
+                                let year = this.x.getUTCFullYear();
+                                let month = this.x.getUTCMonth();
+                                    month += 1; // adding a month to the UTC months (which are zero-indexed)
+                                let formattedUTCDailyDate = month + '/' + UTCday + '/' + year;
+                                return '<br>Date: <b>'  + formattedUTCDailyDate + '</b><br>Value: <b>' + this.y + ' ft³/s'
+                            }
+                        }
+                    },
+                    turboThreshold: 0, 
+                    type    : 'line',
+                    color   : '#add8f2',
+                    data    : this.formattedDailyFlow,
+                    marker: {
+                        symbol: '',
+                        radius: 3
+                    }
+                },
+                {
+                    name    : 'Annual Peak Streamflow',
+                    tooltip: {
+                        headerFormat:'<b>Peak Annual Flow</b>',
+                        pointFormatter: function(){
+                            if (this.formattedPeakDates !== null){
+                                let waterYear = this.x.getUTCFullYear();
+                                if (this.x.getUTCMonth() > 8) { // looking for dates that have a month beginning with 1 (this will be Oct, Nov, Dec)
+                                    waterYear += 1; // adding a year to dates that fall into the next water year
+                                };
+                                let UTCday = this.x.getUTCDate();
+                                let year = this.x.getUTCFullYear();
+                                let month = this.x.getUTCMonth();
+                                    month += 1; // adding a month to the UTC months (which are zero-indexed)
+                                let formattedUTCPeakDate = month + '/' + UTCday + '/' + year;
+                                return '<br>Date: <b>'  + formattedUTCPeakDate + '</b><br>Value: <b>' + this.y + ' ft³/s</b><br>Water Year: <b>' + waterYear
+                            }
+                        }
+                    },
+                    turboThreshold: 0, 
+                    type    : 'scatter',
+                    color   : 'black',
+                    data    : this.formattedPeakDates,
+                    marker: {
+                        symbol: 'circle',
+                        radius: 3
+                    }
+                },
+                {
+                    name    : 'Annual Peak Streamflow (Date Estimated)',
+                    tooltip: {
+                        headerFormat:'<b>Peak Annual Flow</b>',
```

Comment:
Let's call this "Annual Peak Streamflow" so that it matches what is listed in the legend.